### PR TITLE
[IMP] sale_(project),hr_timesheet: generic UX improvement for project

### PR DESCRIPTION
**Pull request by roku-odoo**
Purpose of this commit to improve generic usage of project app.

So in this commit done the following changes:
- the sequence field should not be optional for project.task.type list view
- move the fa-play icon next to the remaining hours widget in kanban view
- add the filters that are highlighted to the project sharing search view
- add group bys for project sharing search view
- add filter and separator for burndown chart search view
- transform the 'milestones' stat button into 'x Milestones /n y Reached
- change the recurrence banner
- add the placeholder for the label_tasks field 'e.g. Tasks'

task-2947481

### CODE REVIEW

**Review by auon-odoo — APPROVED** [review]
Thank you :fire:

File: addons/hr_timesheet/models/project.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
     'subtask_planned_hours',
     'timesheet_ids',
     'total_hours_spent',
+    'remaining_hours_percentage',
```
Comment (xavierbol) [inline addons/hr_timesheet/models/project.py:24]: Why is it needed? 🤔

File: addons/project/models/project.py [hunk 2]
```diff
@@ -53,6 +53,7 @@
     'dependent_ids',
     'is_blocked',
     'message_is_follower',
+    'date_last_stage_update',
```
Comment (xavierbol) [inline addons/project/models/project.py:56]: Why is it needed? 🤔

**Comment by xavierbol** [comment]
Be careful when you will resolve the conflicts, it is probably because we split the files to have one model per file. 🙂

File: addons/hr_timesheet/models/project.py (hunk 1 above)
Comment (ppr-odoo) [reply addons/hr_timesheet/models/project.py]: > Why is it needed? thinking

I checked it but its not used anywhere so I removed.thnaks

File: addons/project/models/project.py (hunk 2 above)
Comment (ppr-odoo) [reply addons/project/models/project.py]: This is given in the write list so now it is not needed in read list.so i removed. [link](https://github.com/odoo/odoo/blob/master/addons/project/models/project_task.py#L51) thanks

**Comment by xavierbol** [comment]
robodoo r+

**Pull request closed without merging**